mysql-test/suite/galera/r/galera_bf_abort.result

```diff
@@ -4,7 +4,7 @@ START TRANSACTION;
 INSERT INTO t1 VALUES (1);
 INSERT INTO t1 VALUES (1);
 INSERT INTO t1 VALUES (2);
-ERROR 40001: Deadlock found when trying to get lock; try restarting transaction
+ERROR 40001: WSREP cancelled the transaction
```

Comment:
We may need to capture the reason.

So something like  "WSREP detected deadlock/conflict. Transaction aborted. Trying restarting"
"dealock/conflict" -> reason that can be replaced if we have specific reason know while raising the error.